runtime/src/main/java/org/corfudb/runtime/BootstrapUtil.java

```diff
@@ -51,10 +52,10 @@ public static void bootstrap(@Nonnull Layout layout,
      * @param retries                Number of retries to bootstrap each node before giving up.
      * @param retryTimeout           Duration between retries.
      */
-    public static void bootstrap(@Nonnull Layout layout,
-                                 @Nonnull CorfuRuntimeParameters corfuRuntimeParameters,
+    public static void bootstrap(@NonNull Layout layout,
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 20 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)